src/Microsoft.DotNet.Configurer/NuGetCachePrimer.cs

```diff
@@ -17,31 +17,20 @@ public class NuGetCachePrimer : INuGetCachePrimer
 
         private readonly INuGetCacheSentinel _nuGetCacheSentinel;
 
-        private readonly CliFolderPathCalculator _cliFolderPathCalculator;
-
-        public NuGetCachePrimer(
-            INuGetPackagesArchiver nugetPackagesArchiver,
-            INuGetCacheSentinel nuGetCacheSentinel,
-            CliFolderPathCalculator cliFolderPathCalculator)
-            : this(nugetPackagesArchiver,
-                nuGetCacheSentinel,
-                cliFolderPathCalculator,
-                FileSystemWrapper.Default.File)
+        public NuGetCachePrimer(INuGetPackagesArchiver nugetPackagesArchiver, INuGetCacheSentinel nuGetCacheSentinel)
```

Comment:
Why the all the formatting changes in the PR?

Reply:
🤷‍♂️ no reason other than personal preference for not wrapping lines that aren't long.